netdata-installer.sh

```diff
@@ -334,7 +334,7 @@ echo >&2
 # run netdata
 
 echo >&2 "Starting netdata..."
-run ${NETDATA_PREFIX}/usr/sbin/netdata "${@}"
+run ${NETDATA_PREFIX}/usr/sbin/netdata -pidfile /var/run/netdata.pid "${@}"
```

Comment:
I am not sure I get the question.
The whole idea of this script is to help users install it quickly. If you look at the blogs, this little thing made a difference. When netdata will be in all distros, this script will be useless. So we need it just to let everyone install it in the mean time.